modules/gltf/src/gltf/gltf-instantiator.ts

```diff
@@ -130,17 +130,19 @@ export class GLTFInstantiator {
   }
 
   createPrimitive(gltfPrimitive: any, i: number, gltfMesh): ModelNode {
```

Comment:
Q: Is there a way for us to get typings on gltfPrimitive?  Having it as `any` feels fragile. `@loaders.gl/gltf` exports extensive typing for both raw and processed gltf. Though of course if that is the only reason we need to import it then the case is weaker.